app-privacy-policy.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Minetest Privacy Policy
+description: Privacy policy for the Minetest application.
+small: |
+  Last updated: 2023-05-10
+  (<a href="https://github.com/minetest/minetest.github.io/commits/master/app-privacy-policy.md">View updates</a>)
+layout: page_subtitle
+---
+
+<style>
+	h2 {
+		margin-top: 5rem !important;
+	}
+</style>
+
+Minetest may connect to the following services during its operation:
+
+* The main website (www.minetest.net): used to get the most recent version
+* The server list: used to load the server list in "Play Online"
+* ContentDB: used to power the Browse Online Content dialog
+* Game servers: third-party servers when playing online
+
+Note: this policy only applies to the Minetest application, accessing the
+mentioned services directly using a web browser is not covered.
+
+### Table of contents
+
+- [Version checking](#version-checking)
+	- [Information collected](#information-collected)
+	- [How it is used](#how-it-is-used)
+- [Loading server list](#loading-server-list)
+	- [Information collected](#information-collected-1)
+	- [How it is used](#how-it-is-used-1)
+	- [Who has access](#who-has-access)
+	- [Period of retention](#period-of-retention)
+	- [Location](#location)
+- [Browsing Online Content (ContentDB)](#browsing-online-content-contentdb)
+	- [Information collected](#information-collected-2)
+	- [How it is used](#how-it-is-used-2)
+	- [Who has access](#who-has-access-1)
+	- [Period of retention](#period-of-retention-1)
+	- [Location](#location-1)
+	- [Removal requests](#removal-requests)
+- [Online play](#online-play)
+- [Third parties](#third-parties)
+- [Future changes to privacy policy](#future-changes-to-privacy-policy)
+
+
+
+## Version checking
+
+### Information collected
+
+When you open Minetest, it may contact www.minetest.net to fetch information
+about the most recent version. This HTTP request will contain the following
+information:
+
+* IP address
+* Minetest version
+* Platform, OS, device, etc
+
+### How it is used
+
+The version information is hosted on GitHub pages, we do not log requests.
```

Comment:
```suggestion
The version information is hosted on GitHub pages, which does not log requests.
```

Reply:
Do we know this?

Reply:
Could always change it to "The website is hosted on GitHub pages, please refer to their privacy policy."

Reply:
> When a GitHub Pages site is visited, the visitor's IP address is logged and stored for security purposes, regardless of whether the visitor has signed into GitHub or not. For more information about GitHub's security practices, see [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).